build_tools/travis/install.sh

```diff
@@ -28,6 +29,7 @@ else
         -DENABLE_COVERAGE:BOOL=${USE_COVERAGE} \
         -DENABLE_PACKAGING:BOOL=${ENABLE_PACKAGING} \
         -DUSE_VTK:BOOL=${USE_VTK} \
+        -DUSE_GIFTI:BOOL=${USE_GIFTI} \
         -DUSE_CGAL:BOOL=${USE_CGAL} \
         -DBLASLAPACK_IMPLEMENTATION:BOOL=${BLASLAPACK_IMPLEMENTATION} \
         -DUSE_SYSTEM_clapack:BOOL=${USE_SYSTEM} \
```

Comment:
OK with the suppression of C casting and the removal of unnecessary {} on for loops.....